README.md

```diff
@@ -92,6 +92,7 @@ The common options are:
 | `type`        | The Cognite Data Fusion resource type. See below for more [resource type examples](#examples-by-resource-types).                                                                                                                                                                                                  | Yes                                        |
 | `maxRetries`  | The maximum number of retries to be made when a request fails. Default: 10                                                                                                                                                                                                                                        |                                            |
 | `maxRetryDelay` | The maximum number of seconds to wait between retrying requests. Default: 30                                                                                                                                                                                                                                        |                                            |
+| `maxParallelRequests` | The maximum number of requests send to CDF in parallel **per partition**. Reduce this number if running into rate limiting issues. Default: 200                                                                                                                                                                                                                                        |                                            |
```

Comment:
This sounds a lot like `DefaultParallelismPerPartition`, perhaps we can use that instead?